### PR TITLE
Update propose-tool issue template

### PR DESCRIPTION
**Pull request by arthurclares**
## 🛠️ Description
This PR updates the template to submit a new tool. 

### 🔬 How did you test this change?

> - [ ] 🤏 Lint tests
> - [ ] 🤞 PS -WhatIf / az validate
> - [ ] 👍 Manually deployed + verified
> - [ ] 💪 Unit tests
> - [ ] 🙌 Integration tests

### 🙋‍♀️ Do any of the following that apply?

> - [ ] 🚨 This is a breaking change.
> - [x] 🤏 The change is less than 20 lines of code.

### 📑 Did you update `docs/changelog.md`?

> - [ ] ✅ Updated changelog (required for `dev` PRs)
> - [ ] ➡️ Will add log in a future PR (feature branch PRs only)
> - [x] ❎ Log not needed (small/internal change)

### 📖 Did you update documentation?

> - [ ] ✅ Public docs in `docs` (required for `dev`)
> - [ ] ✅ Internal dev docs in `src` (required for `dev`)
> - [ ] ➡️ Will add docs in a future PR (feature branch PRs only)
> - [x] ❎ Docs not needed (small/internal change)

### CODE REVIEW

**Review by flanakin — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/propose-tool.md [hunk 1]
```diff
@@ -2,30 +2,79 @@
...
+_Summarize the purpose and functionality of the tool._  
 
-### Additional Information:
-[Include any additional information, links, or resources related to the tool.]
+ Use case:
+_What problem does this tool solve? Who will benefit from it?_  
 
-### Contributor Information:
+### Contributor information:
 - **Name**: [Your Name]
```
[the remark this answers is not in the document]
Comment (arthurclares) [reply .github/ISSUE_TEMPLATE/propose-tool.md:28]: That's a good point. I haven't thought about PII. 
I think if we want to organize a call if the submitter, we can ask them to send an email to the toolkit DL. I will change that.

File: .github/ISSUE_TEMPLATE/propose-tool.md [hunk 2]
```diff
@@ -2,30 +2,79 @@
...
+**Services used:**
+_What are the Azure services needed to deploy this tool?_
+
+**Dependencies:**  
+_List any dependencies, such as other tools, data sources, or teams._  
+
+---
+## 3. Development plan (To be completed by tech lead)
+
+**Tech lead:**
```
[the remark this answers is not in the document]
Comment (arthurclares) [reply .github/ISSUE_TEMPLATE/propose-tool.md:40]: I like onboarding lead.
```suggestion
**Onboarding lead:**  

```

File: .github/ISSUE_TEMPLATE/propose-tool.md [hunk 3]
```diff
@@ -2,30 +2,72 @@
...
+**Dependencies:**  
+_List any dependencies, such as other tools, data sources, or teams._  
+
+### Contributor information:
 - **Name**: [Your Name]
-- **Contact Email**: [Your Email Address]
+
+---
+
+## 2. Development plan (to be completed by tech lead)
```
Comment (arthurclares) [inline .github/ISSUE_TEMPLATE/propose-tool.md:38]:
```suggestion
## 2. Development plan (to be completed by onboarding lead)

```

File: .github/ISSUE_TEMPLATE/propose-tool.md [hunk 4]
```diff
@@ -2,30 +2,79 @@
...
 assignees: ''
 ---
+## Thank You for Contributing!  
+Thank you for taking the time to propose a new tool for the FinOps toolkit. 🙌  
 
-## New Tool Submission
-
-### Tool Name:
-[Insert Tool Name]
+As part of this process, once the tool is merged, you’ll take on the important role of its technical lead. To better understand what this entails, please review the [technical lead responsibilities here](https://github.com/microsoft/finops-toolkit/blob/dev/docs-wiki/Advisory-council.md).
```
[the remark this answers is not in the document]
Comment (arthurclares) [reply .github/ISSUE_TEMPLATE/propose-tool.md:12]: Please look at the yaml file. I sorted it there.

File: .github/ISSUE_TEMPLATE/propose-tool.md [hunk 5]
```diff
@@ -2,30 +2,79 @@
...
+**Dependencies:**  
+_List any dependencies, such as other tools, data sources, or teams._  
+
+---
+## 3. Development plan (To be completed by tech lead)
+
+**Tech lead:**  
+_Name the tech lead overseeing this tool's onboarding._  
+
+**Initial tasks:**
```
[the remark this answers is not in the document]
Comment (arthurclares) [reply .github/ISSUE_TEMPLATE/propose-tool.md:45]: Removed from the YAML

**Pull request merged**